pages/product/index.vue

```diff
@@ -153,9 +219,21 @@ export default defineComponent({
     const options = computed(() => productGetters.getOptions(product.value))
     const addToCart = () => {}
     const changeTab = () => {}
-    const selectOption = async ({ value }, { attributeFQN }) => {
+
+    const selectOption = async ({ value }, { attributeFQN }, className, event) => {
+      applyActiveClass(className, event?.target)
       await configure({ value, attributeFQN }, product.value?.productCode, options.value)
     }
+
+    const applyActiveClass = (className, htmlElement) => {
+      if (htmlElement) {
+        const htmlElementActive = document.getElementsByClassName(className)[0]
+
+        if (htmlElementActive) htmlElementActive.classList.remove(className)
```

Comment:
instead of manipulating the DOM this way, toggle the "active" class in the template by checking the option.value.isSelected property. (Check the useProduct configure method for reference")

Reply:
Resolved